apps/laboratory/src/pages/_app.tsx

```diff
@@ -7,12 +8,22 @@ import type { Session } from 'next-auth'
 
 bootstrapSentry()
 
+declare global {
+  interface Window {
+    Telegram: { WebApp: { ready: () => void } }
+  }
+}
+
 export default function App({
   Component,
   pageProps
 }: AppProps<{
   session: Session
 }>) {
+  useEffect(() => {
+    window.Telegram?.WebApp.ready()
+  }, [])
```

Comment:
is this reqquired? what happens if it fails?

Reply:
This is used by Telegram to initialize the website inside a Mini App. Outside telegram it doesn't have any effect. We can remove this from labs if needed, it's just here for testing purposes.